sdk/identity/azure-identity/azure/identity/_credentials/browser.py

```diff
@@ -43,6 +43,15 @@ class InteractiveBrowserCredential(InteractiveCredential):
         will cache tokens in memory.
     :paramtype cache_persistence_options: ~azure.identity.TokenCachePersistenceOptions
     :keyword int timeout: seconds to wait for the user to complete authentication. Defaults to 300 (5 minutes).
+    :keyword bool allow_broker: Brokers provide single sign-on, device identification, and application identification
```

Comment:
I think this should read `An authentication broker is an application that runs on a user's machine that manages the authentication handshakes and token maintenance for connected accounts.` ....